### ISSUE TITLE
Time field throws an error when editing submission through front end

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#407]
**Description**
When editing a submission on the front end users are getting the following error on the time field if the calendar display type is being used. This doesn't happen when the other display types are used or editing submissions through the back end

<img width="365" alt="Screen Shot 2021-03-23 at 9 29 04 am" src="https://user-images.githubusercontent.com/21274925/112069245-6a177700-8bbb-11eb-8df5-b296049f7141.png">
 
**Steps to reproduce**
1. Create a form with a date/time field using the calendar display type
2. Attempt to edit a submission through a front end form

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.24
- Craft version: 3.6.10
- Multi-site: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.25",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Great, thanks!

**Comment by engram-design** [comment]
Fixed in [1.3.26](https://github.com/verbb/formie/releases/tag/1.3.26)
